images/manageiq-base/container-assets/container_env

```diff
@@ -32,7 +37,7 @@ production:
   encoding: utf8
   pool: 5
   wait_timeout: 5
-  ssl_mode: ${DATABASE_SSL_MODE}
+  ssl_mode: ${database_ssl_mode}
 KEY
 
 [[ -f /.postgresql/root.crt ]] && echo "  sslrootcert=/.postgresql/root.crt" >> ${APP_ROOT}/config/database.yml
```

Comment:
Completely unrelated, so I'll make a different PR, but I think this would be cleaner to append to the database_ssl_mode variable first and then interpolating it into the yml content as opposd to writing the yaml and then appending the file.